mappings/net/minecraft/entity/DirigibleWithItem.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_4981 net/minecraft/entity/DirigibleWithItem
+	METHOD method_26313 travel (Lnet/minecraft/class_1308;Lnet/minecraft/class_4980;Lnet/minecraft/class_243;)Z
+		ARG 1 entity
+	METHOD method_26314 interactMob (Lnet/minecraft/class_1308;Lnet/minecraft/class_1657;Lnet/minecraft/class_1268;Z)Z
+		ARG 1 entity
+		ARG 2 player
+		ARG 3 hand
```

Comment:
not interact but like install saddle. arg4 childAllowed

Reply:
not at all, it also checks if the item is a name tag

Reply:
but arg 4 is still like `childRideable`